### DISCUSSION TITLE
Want to Add fixed sized height of canvas

### DISCUSSION THREAD

**Discussion by amansharmaagami** [GrapesJS/grapesjs#3724] [Q&A]
I'm using it for making dynamic PDF content.
Wanted to fixed the height of canvas so user can't add more blocks from given size (height).

I want to be remove scroll and a message or restriction like can't add more items or something like that.

Found a old issue on same
https://github.com/artf/grapesjs/issues/1803

If it is costly update then please suggest me how to achieve the same.

**Comment by amansharmaagami** [comment]
@artf 

Thanks for making GrapesJS.

**Reply by Temepest74** [reply]
Did you find any sollution?
